### ISSUE TITLE
Compiling Win64 bit Python27 version of Pythonocc

### ISSUE THREAD

**Issue by chenkianwee** [tpaviot/pythonocc-core#115]
I need a win64 python27 pythonocc. I followed the instructions and try to compile a 64bit version, however I am receiving this error. 

![error](https://cloud.githubusercontent.com/assets/13062600/8401618/b7715924-1e63-11e5-96fe-39fd6d789698.png)

Has anyone had this problem before while trying to compile a 64bit version ?

**Comment by tpaviot** [comment]
64 or 32 bit should not be the problem. I sue to compile pythonocc using mingw32 on windows, not msvc. According to the screenshot, it seems TKernel library is not found. Did you download oce-0.16 windows precompiled binaries ? can you please rather test to compile in Release mode ?

**Comment by chenkianwee** [comment]
Thanks for the reply. I compiled my own oce-0.16, let me download the precompiled binaries and try it out with mingw32, lets hope I will be able to succeed this time.

**Comment by chenkianwee** [comment]
Hi, 
I again attempt to build the 64bit version. First of all this time I downloaded the precompiled binaries of oce 0.16.0
1.) So I tried to build the project with msvc, considering with the new oce precompiled binaries it will be able to find all the necessary files this time. But it failed. 
![error1](https://cloud.githubusercontent.com/assets/13062600/8494408/6e887044-2197-11e5-89be-73a5035222ae.png)

2.) So I decided to build it with mingw, so I cmake pythonocc core for mingw compiler.
![mingwmakefile](https://cloud.githubusercontent.com/assets/13062600/8494438/9d7aa322-2197-11e5-8cf5-454ad7b25f70.png)

3.) It manages to generate the files with an error of "error making directory src/vizualisation"
4.) From here i try to compile the files with make command from the msys.bat file, but nothing happen. 
![cmake_make](https://cloud.githubusercontent.com/assets/13062600/8494460/cdf9d180-2197-11e5-942e-f26c58c5f161.png)

Am I doing something wrong here? Is there a guide I can look at to compile the pythonocc? Sorry to be so long-winded
